docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+blah blah some intro
+
+## Prerequisites
+
+Before you begin, understand that this article assumes that you are using Windows and Visual Studio.
```

Comment:
I'd be a little less aggressive in the wording here, such as "This article will be using Visual Studio on Windows." Maybe that would be the right place to say how eventually those scenarios will be doable cross-platform.